<launch>
	<node name="rebvo_node" pkg="rebvo" type="rebvo" output="screen" >
		<rosparam command="load" file="$(find rebvo)/config/rebvo_rosario.yaml"/>

        <param name="rebvo/imu_topic" value="/imu"/>
        <param name="rebvo/image_topic" value="/stereo/left/image_raw"/>
        <param name="rebvo/frame_id" value="rebvo_frame"/>
        </node>

</launch> 
